examples/decoding/plot_decoding_xdawn_transformer.py

```diff
@@ -40,9 +40,10 @@
                 add_eeg_ref=False, verbose=False)
 
 X = epochs.get_data()
-y = label_binarize(epochs.events[:, 2], classes=[1, 3])
+y = label_binarize(epochs.events[:, 2], classes=[1, 3]).reshape(len(X))
```

Comment:
`y` becomes a single column vector which resulted in the error @Eric89GXL mentioned in #3433